airbyte-integrations/connectors/source-snapchat-marketing/source_snapchat_marketing/spec.json

```diff
@@ -4,34 +4,46 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Snapchat Marketing Spec",
     "type": "object",
-    "required": ["client_id", "client_secret", "refresh_token"],
+    "required": ["client_id", "client_secret", "refresh_token", "start_date"],
```

Comment:
won't this break existing configs if they're not setting "start_date"? It has a default value so it's not clear to me why it's also required